adapter/noop/noop.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 type (
```

Comment:
I think this is just for testing purpose, representing someone writing an adapter that supports a bunch of templates(aspect) that our tests depend on. I think we can leave this as is. BTW, there is a noop2 that is equivalent of this for the template world and I think that has to be hand written because tests depend on it.